Sources/SwiftParser/Expressions.swift

```diff
@@ -317,16 +317,7 @@ extension Parser {
         arena: self.arena
       )
 
-      let rhs: RawExprSyntax?
-      if colon.isMissing {
```

Comment:
Contrary to my first comment, I just had a thought. What do we produce if we have the following?

```
condition ? 1
someOtherVariable
```

I think in this case we should be suggesting to add `: <#expression#>` and not just a standalone colon. To achieve that you might need to revert to the old code and just add an additional check for whether the next token is on the same line or a new line.

Reply:
In the current state of the project, it returns the diagnostic "expected ':' after '? ...' in ternary expression." 
The Swift compiler in Xcode works the same way.
As you mentioned, I think it is appropriate to suggest adding ": <#expression#>" to the code. 
I will proceed to work on it further and add an commit in this PR!